doc/content/getting-started/aws/ecs/troubleshooting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 7
+---
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2714
+Move to generic getting started guide once ready.
+-->
+
+This section contains information to troubleshoot {{% tts %}} ECS deployment.
+
+<!--more-->
+
+### TTS no longer accepting connections from UDP gateways after upgrading to latest version
+
+Cross check the below operations
+
+  - Before the version update, complete the database migration.
+  
+  - If `default` is set to the `DefaultTenantId` in the configuration file, unset the `DefaultTenantId` parameter
+  
+  ```
+  tenancy:
+        default-id: 'default'
+  ```
+  Unset the `DefaultTenantId` as below
+  
+  ```
+  tenancy:
+        default-id: ''
+  ```
```

Comment:
Have you seen an instance where this changed due a version update? If yes that should not happen. Please file an internal issue for Michal.

Reply:
Ref: https://github.com/TheThingsIndustries/lorawan-stack-support/issues/523

Reply:
Ok but this advice is still incorrect. If this happens, it's a bug in the templates and customers should reach out to us. Since we've seem to have fixed this, this (hopefully) won't happen again so it can be removed.

Reply:
Okay, @KrishnaIyer.

Do you consider closing this PR?

Reply:
Yes please do.